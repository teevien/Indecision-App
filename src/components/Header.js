import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <div>
        <h1>
            Expensify
  </h1>
        <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/"
        >
            Home
  </NavLink>
        <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/create"
        >
            Add Expense
  </NavLink>
        <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/help"
        >
            Help
  </NavLink>
    </div>
);

export default Header;