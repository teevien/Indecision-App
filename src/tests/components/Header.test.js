// react test renderer
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';
// import toJSON from 'enzyme-to-json'

test('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find('h1').text()).toBe('Expensify');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
});