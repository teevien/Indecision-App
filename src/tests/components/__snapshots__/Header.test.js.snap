// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/create"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
